spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayProperties.java

```diff
@@ -72,6 +73,41 @@
 	 */
 	private List<String> initSqls = Collections.emptyList();
 
+	/**
+	 * Set filename prefix for scripts.
+	 */
+	private String sqlMigrationPrefix = "V";
+
+	/**
+	 * Set filename suffix for scripts.
+	 */
+	private String sqlMigrationSuffix = ".sql";
+
+	/**
+	 * Enable validation when running migrate.
+	 */
+	private boolean validateOnMigrate = true;
+
+	/**
+	 * @see Flyway#baselineOnMigrate
```

Comment:
Don't do that please, the meta-data generation process does not handle javadoc tags.